src/ploneintranet/suite/tests/acceptance/search.robot

```diff
@@ -80,11 +80,6 @@ I can follow the search result ${SEARCH_RESULT_TITLE}
     Click Link  link=${SEARCH_RESULT_TITLE}
     Page should contain  ${SEARCH_RESULT_TITLE}
 
-I can exclude content of type ${CONTENT_TYPE}
-    Unselect Checkbox  css=input[type="checkbox"][value="${CONTENT_TYPE}"]
-    Select From List By Value  css=select[name="created"]  today
-    Wait Until Element is Visible  css=dl.search-results[data-search-string*="created=today"]
-
```

Comment:
Can I ask why this keyword was removed? It now means that the main robot test doesn't read like a user story. Abstracting the actual interactions into a keyword means that when the UI implementation changes, the keywords can be updated and the test cases can stay the same.